test/WebSites/ActionConstraintsWebSite/Startup.cs

```diff
@@ -9,18 +9,19 @@ namespace ActionConstraintsWebSite
 {
     public class Startup
     {
-        public void Configure(IApplicationBuilder app)
+        // Set up application services
+        public void ConfigureServices(IServiceCollection services)
         {
-            var configuration = app.GetTestConfiguration();
-
-            app.UseServices(services =>
+            services.AddMvc();
+            services.ConfigureMvc(options =>
             {
-                services.AddMvc();
-                services.Configure<MvcOptions>(options =>
-                {
-                    options.AddXmlDataContractSerializerFormatter();
-                });
+                options.AddXmlDataContractSerializerFormatter();
             });
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            var configuration = app.GetTestConfiguration();
```

Comment:
triage rejected the rename bug you had me file.  but you're moving an unused variable around and I wasn't suggesting additional changes to lines you weren't already touching.